public/modules/base/services/Role.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+/* Services */
+angular.module('mean').service('roleService', function ($http) {

```

Comment:
moved issue to a differnt task: https://github.com/MakingSense/mean-seed/issues/31, I think it needs a general refactor